src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -160,6 +166,7 @@ private void configMotors() {
 
     @Override
     public void periodic() {
+        System.out.println(targetRPM);
```

Comment:
Please don't just print numbers.  At least print a label so you can distinguish it from all the other numbers printed...